package.json

```diff
@@ -23,6 +23,7 @@
     "@types/prettier": "2.7.2",
     "@vitejs/plugin-react": "4.0.1",
     "eslint": "8.39.0",
+    "glob": "10.3.4",
```

Comment:
@benjroy let's make sure we use the latest version of `glob` now - `10.3.10`: https://www.npmjs.com/package/glob